cinema_service/settings.py

```diff
@@ -47,6 +49,8 @@
     "debug_toolbar",
     "cinema",
     "user",
+    "drf_spectacular",
+    "rest_framework_simplejwt",
```

Comment:
I recommend listing external packages first in the INSTALLED_APPS and then your own apps.